### PR TITLE
Breaking changes with markdown files

### PR DESCRIPTION
**Pull request by rpetrusha**

### CODE REVIEW

File: docs/core/compatibility/2.2-3.0.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: Breaking changes, Version 2.2 to 3.0
```
Comment (mairaw) [inline docs/core/compatibility/2.2-3.0.md]:
```suggestion
title: Breaking changes, version 2.2 to 3.0
```

Comment (mairaw) [reply docs/core/compatibility/2.2-3.0.md]: Do you also want to add a suffix title to this folder to display the .NET Core brand?

File: docs/core/compatibility/2.2-3.0.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+description: Lists the breaking changes from version 2.2 to version 3.0 of .NET Core, ASP.NET Core, and EF Core.
+ms.date: "08/13/2019"
```
Comment (mairaw) [inline docs/core/compatibility/2.2-3.0.md]:
```suggestion
ms.date: "09/05/2019"
```

File: docs/core/compatibility/2.2-3.0.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+author: "rpetrusha"
+ms.author: "ronpet"
+---
+
+# Breaking changes for migration from Version 2.2 to 3.0
```
Comment (mairaw) [inline docs/core/compatibility/2.2-3.0.md]:
```suggestion
# Breaking changes for migration from version 2.2 to 3.0
```

File: docs/core/compatibility/2.2-3.0.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+[!INCLUDE[versionselector](~/includes/core-changes/versionselector.md)]
+
+<!-- [!INCLUDE[introduction](../../../../includes/migration-guide/retargeting/introduction.md)] -->
+
+If you are migrating from version 2.2 to version 3.0 of .NET Core, ASP.NET Core, or EF Core, review the following topics for breaking changes that may affect your app:
```
Comment (mairaw) [inline docs/core/compatibility/2.2-3.0.md]:
```suggestion
If you're migrating from version 2.2 to version 3.0 of .NET Core, ASP.NET Core, or EF Core, review the following topics for breaking changes that may affect your app:
```

File: docs/framework/install/net-framework-needed.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
```
Comment (mairaw) [inline docs/framework/install/net-framework-needed.md]: shouldn't this be article be in a different PR?

File: includes/core-changes/c-locale-maps-to-invariant-locale.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+### Category
+
+- Globalization 
+
+### Affected APIs
+
+All collation and culture APIs is affected by this change.
+
+<!--
```
Comment (mairaw) [inline includes/core-changes/c-locale-maps-to-invariant-locale.md]: do you need this?

File: includes/core-changes/jsonfactoryconverter-createconverter.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    public abstract class JsonConverterFactory : JsonConverter
+    {
+        protected abstract JsonConverter CreateConverter(Type typeToConvert);
+    }
+}
+```
+
+In .NET Core 3.0 Preview 8 and later versions, it is:
+
+``` C#
```
Comment (mairaw) [inline includes/core-changes/jsonfactoryconverter-createconverter.md]:
```suggestion
```csharp
```

File: includes/core-changes/vbnewline-is-obsolete.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
+﻿Public Class Reflectors
```
Comment (mairaw) [inline includes/core-changes/vbnewline-is-obsolete.md]: add vb identifier here?

**Review by mairaw — COMMENTED** [review]
Some quick comments

File: docs/framework/install/net-framework-needed.md (hunk 5 above)
Comment (rpetrusha) [reply docs/framework/install/net-framework-needed.md]: Yes, it should be. I've removed it.

File: includes/core-changes/c-locale-maps-to-invariant-locale.md (hunk 6 above)
Comment (rpetrusha) [reply includes/core-changes/c-locale-maps-to-invariant-locale.md]: Yes. It's for use by tooling.

File: docs/core/compatibility/2.2-3.0.8.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+# Breaking changes for migration from 2.2 to 3.0 Preview 8
+
+> [!IMPORTANT]
+> This article is under construction. This is not a complete list of .NET Core breaking changes. For more information on .NET Core breaking changes, you can examine individual [breaking changes issues](https://github.com/dotnet/docs/issues?q=is%3Aissue+is%3Aopen+label%3Abreaking-change) in the dotnet/docs repo on GitHub. 
+
+If you are migrating from version 2.2 to 3.0 Preview 8 of .NET Core, ASP.NET Core, or EF Core, review the following topics for breaking changes that may affect your app:
+
+## CoreFx
+
+[!INCLUDE[JSonFactoryConverter.CreateConverter signature changed](~/includes/core-changes/jsonfactoryconverter-createconverter.md)]
```
Comment (mairaw) [inline docs/core/compatibility/2.2-3.0.8.md]: Because JSonFactoryConverter was introduced in 3.0, I wouldn't add this information to earlier versions.

**Comment by mairaw** [comment]
We should also add an entry to breadcrumb.

File: includes/core-changes/versionselector.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+> [!div class="op_multi_selector" title1="From target version" title2="To migrated version"]
+> - [(2.2 | 3.0 Preview 8)](~/docs/core/compatibility/2.2-3.0.8.md)
+> - [(2.2 | 3.0)](~/docs/core/compatibility/2.2-3.0.md)
+> - [(3.0 Preview 7| 3.0 Preview 8)](~/docs/core/compatibility/3.0.7-3.0.8.md)
+> - [(3.0 Preview 7| 3.0 ](~/docs/core/compatibility/3.0.7-3.0.md)
```
Comment (mairaw) [inline includes/core-changes/versionselector.md]:
```suggestion
> - [(3.0 Preview 7| 3.0)](~/docs/core/compatibility/3.0.7-3.0.md)
```

**Comment by mairaw** [comment]
I'd double check when the globalization changes were introduced to make sure it's not a change in between previews as well. If it's present in versions earlier than preview 7, I'd probably remove the 2.2 -> 3.0 preview 8 path, since that would be the same as just 3.0.

File: docs/core/compatibility/toc.yml [hunk 11]
```diff
@@ -0,0 +1,8 @@
+- name: Application Compatibility
```
Comment (mairaw) [inline docs/core/compatibility/toc.yml]: interesting that this doesn't render

**Review by mairaw — APPROVED** [review]
LGTM

File: docs/core/compatibility/2.2-3.0.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
+---
+title: Breaking changes, version 2.2 to 3.0 (.NET Core)
```
Comment (mairaw) [inline docs/core/compatibility/2.2-3.0.md]:
```suggestion
title: Breaking changes, version 2.2 to 3.0 - .NET Core
```

File: docs/core/compatibility/3.0.7-3.0.8.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
+---
+title: Breaking changes, version 3.0 Preview 7 to 3.0 Preview 8 (.NET Core)
```
Comment (mairaw) [inline docs/core/compatibility/3.0.7-3.0.8.md]:
```suggestion
title: Breaking changes, version 3.0 Preview 7 to 3.0 Preview 8 - .NET Core
```

File: docs/core/compatibility/3.0.7-3.0.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
+---
+title: Breaking changes, version 3.0 Preview 7 to 3.0 (.NET Core)
```
Comment (mairaw) [inline docs/core/compatibility/3.0.7-3.0.md]:
```suggestion
title: Breaking changes, version 3.0 Preview 7 to 3.0 - .NET Core
```
